### PR TITLE
Bump version to 0.1.0

### PR DESCRIPTION
**Pull request by eb8680**
Blocked by #277 #278 #280

### CODE REVIEW

**Comment by eb8680** [comment]
@agrawalraj can you approve and merge this after #278 merges?

**Comment by agrawalraj** [comment]
@eb8680 yes but I think there's still something wrong with the TOC links for the RNA seq example (fixing that now)

**Review by agrawalraj — APPROVED** [review]

**Pull request merged**
